### PR TITLE
Show result text in job list

### PR DESCRIPTION
**Pull request by chrisvanrun**
Closes: #3495

Re-adds the rendered result text for direct viewing in the job lists. This used to be cramped with the status and results. It is now in its own column. 

The alternative was adding it directly to the results list, but this makes it clearer that it is a singular representation of the result.

Extra CSS was needed to ensure the forcefully added `<p>` in the template rendering does not line out weirdly:

# Pre-CSS
image

# Post-CSS
image

### CODE REVIEW

File: app/grandchallenge/core/static/css/core.css [hunk 1]
```diff
@@ -25,7 +29,8 @@ p {
 }
 
 /* fix modal display with overflow-y: scroll on html */
-.modal, .modal-open {
+.modal,
```
Comment (jmsmkn) [inline app/grandchallenge/core/static/css/core.css:32]: Please undo all of the css formatting changes.

File: app/grandchallenge/core/static/css/core.css [hunk 2]
```diff
@@ -10,6 +10,10 @@ p {
     line-height: 1.6;
 }
 
+.child-mb-0>* {
```
Comment (jmsmkn) [inline app/grandchallenge/core/static/css/core.css:13]: Seems a bit too non-specific? Or is this a problem in many places?

Comment (chrisvanrun) [reply app/grandchallenge/core/static/css/core.css:13]: I thought to use the bootstrap approach and describe the class with what it does. An alternative is to use a very specific 'rendered-result-text' class, but that seemed totally out of place here.

I for see we might want to do this in other places in the future.

File: app/grandchallenge/core/static/css/core.css (hunk 1 above)
Comment (chrisvanrun) [reply app/grandchallenge/core/static/css/core.css]: Mea Culpa, I did not notice. 

It seems like the basic CSS-language-support for VScode has formatting-on-save turned on by default.

File: app/grandchallenge/core/static/css/core.css (hunk 2 above)
Comment (jmsmkn) [reply app/grandchallenge/core/static/css/core.css:13]: Doesn't MarkDown wrap everything in `<p>` so we only need to select that?

Comment (chrisvanrun) [reply app/grandchallenge/core/static/css/core.css:13]: It does. We can make it a bit more specific with `.child-p-md-0>p`.

File: app/grandchallenge/core/static/css/core.css (hunk 1 above)
Comment (jmsmkn) [reply app/grandchallenge/core/static/css/core.css]: Okay we can do this with pre-commit now.

**Review by jmsmkn — APPROVED** [review]

**Pull request merged**